scripts/common/client.py

```
@@ -0,0 +1,1087 @@
+            result[tag] = match.group(1)
+    return result
+
+
+def _parse_token_response(response: httpx.Response) -> dict:
+    """Parse token response, handling both JSON and XML formats."""
+    try:
+        data = response.json()
+        if isinstance(data, dict) and ("AccessToken" in data or "access_token" in data):
+            return data
+    except Exception:
+        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)